frontend/src/components/auth/Login.tsx

```diff
@@ -47,45 +69,85 @@ const Login = (): React.ReactElement => {
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
-          >
-            Log In
-          </button>
-        </div>
-      </form>
-      <div>
-        <button
-          className="btn btn-primary"
-          type="button"
-          onClick={onSignUpClick}
-        >
-          Sign Up
-        </button>
-      </div>
-    </div>
+    <Flex>
+      <Center flex="1">
+        <VStack>
+          <Image height="13vh" marginBottom="0.5rem" src={logo} />
+          <Text variant="display-sm-sb">Sign in to access courses</Text>
+          <form>
+            {isSecondPage ? (
+              <>
+                <FormControl>
+                  <Text 
+                    variant="caption-md"
+                    marginTop="1.5rem"
+                    > Password</Text>
+                  <Input
+                    type="password"
+                    placeholder="●●●●●"
+                    marginBottom="1rem"
+                    onChange={(event) => setPassword(event.target.value)}
```

Comment:
For some reason, when I go through the form login, it auto-populates the password which a bunch of characters. Not sure why this is, but it'd be good to get rid of. It's not the placeholder, but something is there that I have to delete manually. Are other people experiencing this?
![image](https://user-images.githubusercontent.com/41529410/160035059-9ec359c5-bbd5-4957-a4a6-1e77e3892d47.png)

Reply:
This one's a bit weird, when I enter 4 characters in the email, there are 4 dots; same for other #s. Is it maybe picking up on the same input field?